Formula/memcacheq.rb

```diff
@@ -17,7 +18,7 @@ class Memcacheq < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "42513283d3c7b3f42678cc887b73446cc2b216bda5feb2c1a8d1ab0b47d352e8"
   end
 
-  depends_on "berkeley-db"
+  depends_on "berkeley-db@5"
```

Comment:
May want to add a comment on licensing as reason for downgrade.

Reply:
Agreed, though I should probably do that for other formulas with the downgrade. I'll add the comments for other formulae in a separate PR.